Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
```

Comment:
> You are allowed to go up to 120 characters, you know ;) 
Can we add some check operation for this in tests, maybe in `StyleCI`? I always forget about it(((